src/dpos-user.ts

```diff
@@ -160,12 +170,20 @@ export class DPOSUser {
     this._ethereumGateway = new ethers.Contract(gatewayAddress, gatewayABI, wallet)
     // @ts-ignore
     this._ethereumLoom = new ethers.Contract(loomAddress, ERC20ABI, wallet)
+    if (version === GatewayVersion.MULTISIG && vmcAddress !== undefined) {
+      // @ts-ignore
+      this._ethereumVMC = new ethers.Contract(vmcAddress, ValidatorManagerContractABI, wallet)
```

Comment:
If the `vmcAddress` is giving an error on the build here maybe is because it's saying that it can be `undefined`, right? In this case, instead of @ts-ignore, you can use like `vmcAddress!` it's a way to tell to the compiler that it can't be null

Reply:
this is fine as is, no error by the compiler